flink-connector-mongodb-cdc/src/test/java/com/ververica/cdc/connectors/mongodb/table/MongoDBTableFactoryTest.java

```diff
@@ -164,7 +163,7 @@ public void testMetadataColumns() {
 
         MongoDBTableSource expectedSource =
                 new MongoDBTableSource(
-                        TableSchemaUtils.getPhysicalSchema(fromResolvedSchema(SCHEMA)),
+                        SCHEMA_WITH_METADATA,
```

Comment:
@leonardBang I think it's a bug before.

Reply:
@Jiabao-Sun Can you help confirm this?

Reply:
Yes, I think it's a typo here, and the PR #780 can help avoid this issue.